content/applications/studio/models_modules_apps.rst

```diff
@@ -212,16 +212,87 @@ Export and import customizations
 ================================
 
 When you do any customization with Studio, a new module named :guilabel:`Studio customizations` is
-added to your database.
+added to your database. Exporting customizations from one database and importing them in another can
+be useful, for example, when setting up a new module or for training purposes.
 
-To export these customizations, go to :menuselection:`Main dashboard --> Studio --> Customizations
---> Export` to download a ZIP file containing all customizations.
+.. note::
+
+   Exporting customizations from Studio is different from the standard Odoo export function, which
+   allows you to export data as an Excel or csv file. With :guilabel:`Studio Export`, data is
+   exported as a ZIP file, which corresponds to a module.
+
+   When the module is imported in another database, the data is imported in a logical way, e.g. if
+   the module contains customers and sales orders, the customers are created first, since these are
+   required for the sales orders.
+
+.. _studio/export-import/export:
+
+Export customizations
+---------------------
+
+To export customizations, from the main Odoo dashboard click the **Toggle Studio** button, then
+:guilabel:`Export`.
+
+You can download a ZIP file containing all Studio customizations by clicking the :guilabel:`Export`
+button. Alternatively, you can choose which data and/or demo data to export by clicking
+:guilabel:`Configure data and demo data to export`.
+
+.. _studio/export-import/export/configure:
+
+Configure data to export
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To select specific models to export, on the :guilabel:`Studio Export` screen click
+:guilabel:`New`, then choose the relevant model.
+
+.. tip::
+
+   Click :guilabel:`Preset` to see a list of all models in your database with records that have been
+   modifed using Studio. To configure one of these models for export, click on the model to open it
+   and make the required changes.
+
+Tick the following options as relevant:
+
+- :guilabel:`Demo`: if the exported records should be considered as demo data when imported.
+- :guilabel:`Attachments`: if attachments related to exported records should be included in the
+  exported.
+- :guilabel:`Updatable`: if the exported records should be able to updated during a module update.
 
-To import and install these customizations in another database, connect to the destination database
-and go to :menuselection:`Main dashboard --> Studio --> Customizations --> Import`, then upload
-the exported ZIP file before clicking on the :guilabel:`Import` button.
+To set or change the rules determining which of the model's records should be exported, click the
+:icon:`fa-caret-right` icon next to :guilabel:`Domain` then click :guilabel:`Edit domain`.
```

Comment:
Clicking the carret-right icon is only needed when reopening something that was previously created. Still, it needs to be explained as the UI/behaviour can be confusing.
![image](https://github.com/user-attachments/assets/88a70c4f-08a6-4361-bbd5-20e597528458)